apps/meteor/app/apps/server/communication/rest.js

```diff
@@ -560,6 +560,30 @@ export class AppsRestApi {
 			},
 		);
 
+		this.api.addRoute(
+			'/featured',
+			{ authRequired: false },
+			{
+				get() {
+					const baseUrl = orchestrator.getMarketplaceUrl();
+
+					let result;
+					try {
+						result = HTTP.get(`${baseUrl}/v1/apps/featured`);
```

Comment:
Let's include the workspace token, if it exists, and also include the framework version in the request.